cloudformation_templates/app_base.j2

```diff
@@ -281,6 +282,74 @@
       }
     },
 
+    {{ metrics.metric_filter("HTTP5xxMetricFilter",
+                             pattern="$.logType = apache-access && $.status = 5*",
+                             metric_name="HTTP5xx",
+                             metric_value=1) }},
+
+    {{ metrics.metric_filter("HTTPNon5xxMetricFilter",
+                             pattern="$.logType = apache-access && $.status != 5*",
+                             metric_name="HTTP5xx",
+                             metric_value=0) }},
+
+    {{ metrics.alarm("HTTP5xxCountAlarm",
+                    metric_name="HTTP5xx",
+                    statistic="Sum",
+                    period=60,
+                    evaluation_periods=1,
+                    threshold=10,
```

Comment:
500s feel like something we could alert on with fewer than ten - in that we shouldn't ever see them and if we do then the app might need to be fixed.  So maybe even just one as a threshold?